data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,245 @@ mission "A Sunracer Hovercraft Racing"
+			`	And flips your chosen, decidedly non-ace card. You've lost 1,000 credits this round.`
+			`	Then he playfully reveals the left card: the ace.`
+				to display
+					or
+						and
+							"monte: card spot" == 0
+							"monte: card spot" != "monte: guess"
+						and
+							"monte: card spot" != 0
+							"monte: card spot" == "monte: guess"
+			`	With a single, clean motion he shows that the middle card was the ace.`
+				to display
+					or
+						and
+							"monte: card spot" == 1
+							"monte: card spot" != "monte: guess"
+						and
+							"monte: card spot" != 1
+							"monte: card spot" != 2
+							"monte: card spot" == "monte: guess"
+			`	He then sweeps his hand to reveal that the right card was the ace.`
+				to display
+					and
+						"monte: card spot" == 2
+						"monte: card spot" != "monte: guess"
+			branch scammed
+				"monte: card spot" == "monte: guess"
+			`	"Better luck next time."`
+				goto again
+			label scammed
+			action
+				"monte: scammed" = 1
+			`	The dealer shrugs, "Better luck next time."`
+				goto again
+			label correct
+			`	And flips over an ace! The dealer seems bewildered, but begrudgingly hands over 2,000 credits, doubling your money.`
+			action
+				payment 2000
+			label again
+			branch cutoff
+				"monte: games played" < 3
+			`	The dealer looks spooked by some unseen authorities and shoos you away, quickly packing up and disappearing into the spaceport.`
+				goto trueending
+			label cutoff
+			`	The dealer gives you a look. "Fancy another game?"`
+			action
+				"monte: games played" ++
+			choice
+				`	"Of course!"`
+					to display
+						and
+							credits > 1000
+					goto newgame
+				`	"You cheated!"`
+					to display
+						"monte: scammed" >= 1
+				`	"Not today."`
+					goto trueending
+			branch pirateworld
+				"flagship planet attribute: pirate" > 0
+			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
+			choice
+				`	"Fine. But I'm watching you."`
+					to display
+							credits > 1000
+					goto newgame
+				`	"And get scammed again? I don't think so."`
+					goto trueending
+			label pirateworld
+			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`
+			`	From the crowd, several burly figures step forward, weapons on hips, while the dealer's grin never fades.`
+			`	"Matter of fact, you've ruined the game. That's a mark's fee, 10,000 credits. Hand it over." He jerks a thumb at the hulking men circling close. "They'll collect if you're shy."`
+			choice
+				`	"You got it, here's 10,000 credits."`
+					to display
+						credits > 10000
+				`	(Book it for your ship.)`
+					goto bookit
+			`	Your credit chips thud as they land on the table, and you see the dealer's grin widen while you're fleeced and embarrassed.`
```

Comment:
The credits are never actually taken away.

Reply:
Whoops, fixed.